core/model/modx/processors/system/country/getlist.class.php

```diff
@@ -1,37 +1,45 @@
 <?php
+
 /**
  * Gets a list of country codes
  *
  * @package modx
  * @subpackage processors.system.country
  */
-class modCountryGetListProcessor extends modProcessor {
-    public function checkPermissions() {
+class modCountryGetListProcessor extends modProcessor
+{
+    public function checkPermissions()
+    {
         return $this->modx->hasPermission('countries');
     }
 
-    public function process() {
+    public function process()
+    {
         $countryList = $this->getCountryList();
-        if (empty($countryList)) return $this->failure();
+        if (empty($countryList)) {
+            return $this->failure();
+        }
 
         $countries = array();
-        foreach ($countryList as $country) {
+        foreach ($countryList as $iso => $country) {
             $countries[] = array(
-                'value' => $country,
```

Comment:
Maybe it's worthwhile keeping the `value` option here as well for backwards compatibility, just in case some code relies on the value from the processor response?